etc/spack/defaults/darwin/packages.yaml

```diff
@@ -25,3 +25,7 @@ packages:
     # although the version number used here isn't critical
       apple-libunwind@35.3: /usr
     buildable: False
+  python:
+    paths:
+      python@3.7.7: /usr/local/opt/python3
+    version: [3.7.7]
```

Comment:
The brew installed python version will change over time. But currently no better way - unless we run some commands to populate the version in this yaml.

Reply:
Indeed. For Ubuntu we do the same

Reply:
> > ```
> > 3. We have the `sysconfig` directory for other systems, maybe it would good to move that there? @pramodk do you have opinions? …also add a little installation script like for Ubuntu…?
> > ```
> 
> Thanks for the idea @matz-e , I will open a ticket for that

Just for reference:  [BSD-120] spack - create macos documentation + setup script (a la ubuntu)

Reply:
We should get a better way of doing this. We are defaulting to use Python from brew which the user may not have and, moreover, is not compatible with a lot of stuff (remember the wheel?). Plus, version will mismatch soon.
I don't think we should try to be smart here. Users running Spack on mac should know how to add en entry in packages.yaml. Or am I missing the true purpose of this line?

Reply:
It would actually be simplest if there were different python versions - one from Spack, one from brew. Don't mix and match…

Reply:
@ferdonline yes, we should remove that. As @matz-e was suggesting, was thinking about having a sort of setup script like for Ubuntu. On the other hand user should know how to Mac spack you are right :) Maybe just put it in a readme then